dpnp/dpnp_utils/dpnp_utils_linearalgebra.py

```diff
@@ -1812,6 +1833,12 @@ def dpnp_einsum(
             arrays.append(a)
 
     res_usm_type, exec_q = get_usm_allocations(arrays)
+    if out is not None:
+        dpnp.check_supported_arrays_type(out)
+        if dpctl.utils.get_execution_queue((exec_q, out.sycl_queue)) is None:
+            raise ExecutionPlacementError(
```

Comment:
Not sure if intended, but that missed in coverage

Reply:
I see that these lines are reported as not covered in coverage report but they should have been covered  It is covered with the test that was added in `tests/test_linalg.py`.

Reply:
I believe the reason is that there is no any test to cover `out` keyword without raising `ExecutionPlacementError` exception, i.e. the check at line#1838 is always `True` and no test where it is `False`.

Reply:
added a test to resolve this issue. Will add more test for `einsum` function in a separate PR.